spring-petclinic-api-gateway/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+zuul.debug.request=true
+logging.level.org.springframework=DEBUG
```

Comment:
We don’t want to enable debug mode by default. It should be switched on explicitly for debugging purposes.